--- a/rayimg/src/rayimg/imageloader/imageloader.tl	2025-11-18 08:32:05.764026420 +0000
+++ b/rayimg/src/rayimg/imageloader/imageloader.tl	2025-11-18 08:32:09.062054210 +0000
@@ -67,7 +67,7 @@
     local loaded_image: imageinterface.LoadedImage = {}
     
     if cache_directory ~= nil then
-        local cache_location = path.abs(cache_directory) .. filepath .. ".jpg"
+        local cache_location = path.abs(cache_directory) .. "/" .. filepath .. ".jpg"
         if path.isfile(cache_location) then
             print("Loading from cache", cache_location)
             texture = rl.LoadTexture(cache_location)
@@ -88,7 +88,7 @@
         texture = rl.LoadTextureFromImage(loaded_image.image)
         
         if cache_directory ~= nil and loaded_image.should_cache then
-            local cache_file = path.abs(cache_directory) .. filepath .. ".jpg"
+            local cache_file = path.abs(cache_directory) .. "/" .. filepath .. ".jpg"
             local cache_file_dir = path.parent(cache_file)
             fs.makedirs(cache_file_dir)
             rl.ExportImage(loaded_image.image, cache_file)
